src/main/java/it/gov/pagopa/payhub/ionotification/connector/IORestConnectorImpl.java

```diff
@@ -25,7 +31,15 @@ public IORestConnectorImpl(
     }
     @Override
     public ServiceResponseDTO createService(ServiceRequestDTO serviceRequestDTO) {
-        return ioFeignRestClient.createService(serviceRequestDTO, subscriptionKey);
+        try {
+            return ioFeignRestClient.createService(serviceRequestDTO, subscriptionKey);
+        } catch (FeignException e) {
+            log.error("An error occurred while creating service: {}", e.getMessage(), e);
+            if (e.status() == 400){
+                throw new IOWrongPayloadExceptiion(String.format("There is something wrong with the payload: %s", e.getMessage()));
+            }
+            throw new CreateServiceInvocationException("The service was not created, please retry it");
```

Comment:
let's always add the original throwable to the raised exception if they could contain useful info
more over, could you also log some info related to the request?
```suggestion
            throw new CreateServiceInvocationException("The service was not created, please retry it", e);
```